<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IHMCalcIP</class>
 <widget class="QMainWindow" name="IHMCalcIP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>788</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>788</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IHMCalcIP</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="IPEdit1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="IPEdit2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="IPEdit3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="IPEdit4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>30</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>AdresseIP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Suffixe /</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>511</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>RÃ©sultat</string>
    </property>
    <widget class="QLineEdit" name="ResReseau">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ResIPutil">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ResDiffu">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Adresse Reseau</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Adresse IP Utilisables</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Adresse De Diffusion</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="combSuf">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>30</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>10</number>
    </property>
    <property name="maxCount">
     <number>30</number>
    </property>
   </widget>
   <widget class="QComboBox" name="combMask">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="maxCount">
     <number>2147483647</number>
    </property>
   </widget>
   <widget class="QComboBox" name="combSRmax">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combHotesmax">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>190</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Masque du reseau</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre maximum de sous reseaux</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>190</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre d'hotes maximum pour le reseau</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnQuitter">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>BTS SNIR Lycee Malraux Bethune</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnValider">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>IPEdit1</tabstop>
  <tabstop>IPEdit2</tabstop>
  <tabstop>IPEdit3</tabstop>
  <tabstop>IPEdit4</tabstop>
  <tabstop>combSuf</tabstop>
  <tabstop>combMask</tabstop>
  <tabstop>combSRmax</tabstop>
  <tabstop>combHotesmax</tabstop>
  <tabstop>BtnValider</tabstop>
  <tabstop>ResReseau</tabstop>
  <tabstop>ResIPutil</tabstop>
  <tabstop>ResDiffu</tabstop>
  <tabstop>BtnQuitter</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>BtnQuitter</sender>
   <signal>clicked()</signal>
   <receiver>IHMCalcIP</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>679</x>
     <y>538</y>
    </hint>
    <hint type="destinationlabel">
     <x>754</x>
     <y>558</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combSuf</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>combMask</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combSuf</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>combSRmax</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>740</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>632</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combSuf</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>combHotesmax</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>749</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>631</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combHotesmax</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>combSuf</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>632</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>748</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combMask</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>combSuf</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>631</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>713</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combSRmax</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>combSuf</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>637</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>740</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combMask</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>combSRmax</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>475</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combMask</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>combHotesmax</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combHotesmax</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>combMask</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>632</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>632</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combHotesmax</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>combSRmax</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combSRmax</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>combMask</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>542</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combSRmax</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>combHotesmax</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>541</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
